src/collectd.conf.pod

```diff
@@ -1455,6 +1455,20 @@ By default no detailed zone information is collected.
 
 =back
 
+=head2 Plugin C<buddyinfo>
+
+The B<buddyinfo> plugin collects information by reading "/proc/buddyinfo".
+This file contains information about the number of available contagious
+physical pages at the moment.
+
+=over 4
+
+=item B<Zone> I<ZoneName>
+
+Zone to collect info about. Will collect all zones by default.
```

Comment:
Optional: leaving the string empty like `Zone ""` should also result in default behavior.